### DISCUSSION TITLE
Welcome to clojure-test Discussions!

### DISCUSSION THREAD

**Discussion by seancorfield** [clojure-expectations/clojure-test#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Please be polite and respectful at all times -- see the project's [Code of Conduct](https://github.com/clojure-expectations/clojure-test/blob/develop/CODE_OF_CONDUCT.md) 

  As a reminder, the [latest documentation is on cljdoc.org](https://cljdoc.org/d/com.github.seancorfield/expectations/CURRENT).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kkinnear** [comment]
A quick usage report on `2.0.143` (I seem to have missed `2.0.137` unfortunately).  It works without any issue on zprint's 1449 Clojure assertions and 1357 Clojurescript assertions.  Which is not unexpected, but still nice to see.  Thanks for continuing to push expectations forward and bringing it to `2.x`!

**Reply by seancorfield** [reply]
Thanks!
